.gitignore

```diff
@@ -0,0 +1,23 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
```

Comment:
- [ ] You are not following the Gitflow for the review. For the review you are supposed to use Gitflow. Means you are supposed to create a feature branch from development branch and ask for a review from that feature branch 😊

**I know its confusiong for you that i am asking this change but what have you done so far should be done when your project is a group project but as you can see this project is solo. Sio here, you are supposed to use Gitflow 🤔 Let me know if you find any confusion. You are reach out to me on slack by searching my name Hammas in the full-time-students channel 😊**